src/osm_parser.rs

```diff
@@ -93,19 +94,20 @@ pub fn parse_osm_data(
     );
 
     // Determine which dimension is larger and assign scale factors accordingly
-    let (scale_factor_x, scale_factor_z) = if (bbox_scaled.2 - bbox_scaled.0) > (bbox_scaled.3 - bbox_scaled.1) {
-        // Longitude difference is greater than latitude difference
-        (
-            ((bbox_scaled.2 - bbox_scaled.0) * 10 / 100) as f64, // Scale for width (x) is based on longitude difference
-            ((bbox_scaled.3 - bbox_scaled.1) * 14 / 100) as f64, // Scale for length (z) is based on latitude difference
-        )
-    } else {
-        // Latitude difference is greater than or equal to longitude difference
-        (
-            ((bbox_scaled.3 - bbox_scaled.1) * 10 / 100) as f64, // Scale for width (x) is based on latitude difference
-            ((bbox_scaled.2 - bbox_scaled.0) * 14 / 100) as f64, // Scale for length (z) is based on longitude difference
-        )
-    };
```

Comment:
I don't fully understand what this code is for. If we just use scale_factor_* to determine how big the output should be, why does it depend on whether the latitude delta is bigger/smaller than the longitude delta? And what's the difference of 10 and 14?